packages/cli/src/daemon/log-requests.ts

```diff
@@ -33,6 +33,26 @@ export function logRequests(loggerProvider: LoggerProvider): any[] {
     }
     return ' params=-'
   })
+  morgan.token<Request, Response>('body', (req) => {
+    if (req.body) {
+      const keys = Object.keys(req.body)
+      if (keys.length > 0) {
+        const body = keys.reduce((prev, curr) => {
+          const value = req.body[curr]
+          const valKeys = Object.keys(value)
+          if (valKeys.length > 0) {
+            // value is an object
+            return prev + ` body.${curr}=${JSON.stringify(value)}`
```

Comment:
is this going to be extremely long?  agree we want it tho

Reply:
does seem like a possible risk, yes.  This is printing the full body of the commit being sent to js-ceramic, so is users are creating large json documents in their app, this will print the full document, which could be up to 256kb large

Reply:
um if we do this for every single request on gitcoin i definitely think its a risk of crashing them by filling up the logs.  i would like to sample them